base/tps/shared/conf/CS.cfg

```diff
@@ -11,12 +11,21 @@ applet._002=# SAF Key:
 applet._003=# applet.aid.cardmgr_instance=A0000001510000
 applet._004=# Stock RSA,KeyRecover applet : 1.4.58768072.ijc 
 applet._005=# RSA/KeyRecovery/GP211/SCP02, SCP03 applet : 1.5.558cdcff.ijc
-applet._006=# SCP03 AES server side keygen keywrap applet : 1.5.64260792.ijc
-applet._007=# Use GP211 applet only with SCP02 card
-applet._008=# For protocol > 1 do this ex: proto 3 : op.format.userKey.update.applet.requiredVersion.prot.3=1.5.558cdcff, 1.5.64260792.ijc for AES keywrap
-applet._009=# Use existing config for standard proto1 cards ex: op.format.userKey.update.applet.requiredVersion=1.4.58768072
-applet._010=#########################################
-applet.aid.cardmgr_instance=A0000000030000
+applet._006=# SCP03 AES server side keygen keywrap applet :1.5.65cbf5a6.ijc 
+applet._007=# Use GP211 applet only with SCP03 card
+applet._008=# Set the applet for scp03 tokens as follows:
+applet._009=# Examples:  
+applet._010=#Format:
+applet._011=#op.format.userKey.update.applet.requiredVersion.prot.3=1.5.65cbf5a6.ijc
+applet._012=# Enrollment: applet_013=#op.enroll.userKey.update.applet.requiredVersion.prot.3=1.5.65cbf5a6.ijc
```

Comment:
maybe my eyes are deceiving me, but is comment number 13 not having it's own line?  Oh, I think it's missing a '.'

Reply:
Fixing...